packages/web/vite.config.ts

```diff
@@ -7,6 +7,25 @@ export default defineConfig({
   plugins: [
     handlebars({
       partialDirectory: resolve(__dirname, 'partials'),
+      runtimeOptions: {
```

Comment:
Please add some notes to `CONTRIBUTING.md` about places that should developer update when adding a new component. Thanks :-)

Reply:
I've added mention in `File Structure`.